src/main/java/cuchaz/enigma/command/MappingCommandsUtil.java

```diff
@@ -91,7 +91,11 @@ private MappingCommandsUtil() {}
         MappingFormat format = null;
         try {
             format = MappingFormat.valueOf(type.toUpperCase());
-        } catch (IllegalArgumentException ignored) {}
+        } catch (IllegalArgumentException ignored) {
+            if (type.equals("tinyv2")) {
```

Comment:
Maybe move this outside of the `try` and move the `try` to an `else` block?

Reply:
@liach ?